cypress/utils/utils.ts

```diff
@@ -457,6 +457,11 @@ export function applySearchFilter(
     identifiedRisk = false,
     value?: number
 ): void {
+    cy.get("body").then(($body) => {
+        if ($body.find('button:contains("Clear all filters")').length > 0) {
+            clearAllFilters();
+        }
+    });
```

Comment:
I'm not sure this belongs here, I may want to apply multiple search filters without removing the previous ones.

Reply:
True! i moved it to beforeEach test block